### PR TITLE
Removing IE7 test requirement.

### PR DESCRIPTION
**Pull request by javathunderman**
Per issue #1488. Let me know if I need to add/remove anything!

### CODE REVIEW

**Comment by javathunderman** [comment]
Also removing dukgo links.

**Comment by javathunderman** [comment]
@moollaza @MrChrisW @mintsoft Ping!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -28,7 +28,8 @@ If you're submitting a **pull request** (bugfix/addition):
 
 ## Making Changes
 
-- Before making any changes, refer to the [DuckDuckHack Styleguide](https://dukgo.com/duckduckhack/styleguide_overview) to ensure your changes are made in the correct fashion
+- Before making any changes, refer to the [DuckDuckHack Styleguide](https://duck.co
+- /duckduckhack/styleguide_overview) to ensure your changes are made in the correct fashion
```
Comment (MrChrisW) [inline CONTRIBUTING.md]: Looks like this link is broken!

Comment (javathunderman) [inline CONTRIBUTING.md]: @MrChrisW That's weird...Works for me.

Comment (MrChrisW) [inline CONTRIBUTING.md]: @javathunderman Theres an added space after `duck.co` see the display on this [page](https://github.com/javathunderman/zeroclickinfo-spice/blob/patch-1/CONTRIBUTING.md)

selection_233

Comment (javathunderman) [inline CONTRIBUTING.md]: @MrChrisW My mistake, should be fixed now...

**Comment by MrChrisW** [comment]
@javathunderman Just relised there's a dead link on [Line 60](https://github.com/duckduckgo/zeroclickinfo-spice/pull/1526/files#diff-6a3371457528722a734f3c51d9238c13R60) may as well fix it while we're here! Mind updating it to `http://schacon.github.io/gitbook/4_interactive_rebasing.html`
:+1:

**Comment by javathunderman** [comment]
@MrChrisW Fixed it!

**Comment by javathunderman** [comment]
@MrChrisW Also going to get rid of another dukgo link I saw in the readme...

**Comment by jagtalon** [comment]
@javathunderman nice! :+1:

**Pull request merged**